src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -103,6 +104,9 @@ public AddressBook load() throws StorageOperationException {
             throw new StorageOperationException("Error writing to file: " + path);
         } catch (IllegalValueException ive) {
             throw new StorageOperationException("File contains illegal data values; data type constraints not met");
+        } catch (ReadOnlyFileSystemException roe) {
+            throw new StorageOperationException("You have accidentally made the storage file read-only while the " +
+                    "address book was running. ");
```

Comment:
This is for LO W4.10d

Reply:
These are unrelated changes, please refrain from putting them in the same PR in the future.